CityofToronto/bdit_flashcrow
# MOVE-1001: School Crossing Guard hours are wrong

# Issue Addressed
This PR closes [MOVE-1001](https://move-toronto.atlassian.net/browse/MOVE-1001)


![MOVE-1001](https://github.com/CityofToronto/bdit_flashcrow/assets/5003768/080b879f-202c-4912-9bf8-0bcce95b77a8)

# Description
According to [MOVE-1001](https://move-toronto.atlassian.net/browse/MOVE-1001), the Crossing Guard study hours were incorrect. The hours were stated as: 7:30-9:30, 11:00-13:30, 14:30-17:30, but should be 7:30-9:30, 11:00-13:30, 14:30-**17:00**

This change fixes that copy.

# Tests
This was a copy change, so no specific tests were created or ran. To validate this fix follow the Steps to reproduce in [MOVE-1001](https://move-toronto.atlassian.net/browse/MOVE-1001):

- Open the study request form
- Select “School Crossing Guard Study” from the study type dropdown
- Select “School Crossing Guard” hours from the hours dropdown

Verify that the hours are listed as: 7:30-9:30, 11:00-13:30, 14:30-17:00
